playwright-e2e/pages/manage-hearings.ts

```diff
@@ -0,0 +1,140 @@
+import {type Page, type Locator, expect} from "@playwright/test";
+import {BasePage} from "./base-page";
+import {HearingDetailsMixin} from "./mixins/hearing-details-mixin";
+
+export class ManageHearings extends HearingDetailsMixin(BasePage)
+{
+  readonly hearingTypesLabelLocator: Locator;
+  readonly hearingDetails: Locator;
+  readonly hearingDay: Locator;
+  readonly hearingMonth: Locator;
+  readonly hearingYear: Locator;
+  readonly hearingLengthInHours: Locator;
+  readonly hearingLengthInMinutes: Locator;
+  readonly inpPersonCheckbox: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.hearingDetails = this.page.getByLabel('Add details (Optional)');
+    this.hearingDay = this.page.getByRole('textbox', { name: 'Day' });
+    this.hearingMonth = this.page.getByRole('textbox', { name: 'Month' });
+    this.hearingYear = this.page.getByRole('textbox', { name: 'Year' });
+    this.hearingLengthInHours = this.page.getByLabel('Hearing length, in hours');
+    this.hearingLengthInMinutes = this.page.getByLabel('Hearing length, in minutes');
+    this.hearingTypesLabelLocator = this.page.locator('#hearingType .multiple-choice > label');
+    this.inpPersonCheckbox = this.page.getByText('In person');
+  }
+
+  async createNewHearingOnCase(){
+    await this.page.getByLabel('Add a new hearing').check();
+    await this.clickContinue();
+    await this.completeHearingDetails();
+    await this.page.getByRole('radio', { name: 'Yes' }).check();
+    await this.page.getByRole('button', { name: 'Continue' }).click();
+    await this.page.getByRole('radio', { name: 'No' }).check();
+    await this.clickContinue();
+    await this.checkYourAnsAndSubmit();
+  }
+
+  async editPastHearingOnCase() {
+    await this.page.getByLabel('Edit a hearing that has taken').check();
+    await this.page.getByLabel('Which draft hearing do you').selectOption('1: f2be08a2-4daf-4aa3-b7ba-95843b4bcb88');
+    await this.clickContinue();
+    await this.page.getByRole('radio', { name: 'Yes' }).check();
+    await this.clickContinue();
+    await this.checkYourAnsAndSubmit();
+  }
+
+  async vacateHearing() {
+    await this.page.getByLabel('Vacate a hearing - the').check();
+    await this.page.locator('#vacateHearingDateList').selectOption('2: f2be08a2-4daf-4aa3-b7ba-95843b4bcb88');
```

Comment:
Instead of "2: f2be08a2-4daf-4aa3-b7ba-95843b4bcb88" use the hearing details as seen on the screen to select the value.